src/menus/TrackMenus.cpp

```diff
@@ -67,9 +67,9 @@ void DoMixAndRender
 
       // But before removing, determine the first track after the removal
       auto last = *trackRange.rbegin();
-      auto insertionPoint = * ++ tracks.Find( last );
+      auto insertionPoint = * ++ tracks.FindLeader( last );
```

Comment:
Because, operator ++ behaves differently, iterating leaders only and not channels.

This change makes `insertionPoint` correct.  Without it, just making a new empty stereo track, and mixing-and-rendering, corrupts the track list and causes the crash.